src/v/cloud_storage/topic_mount_handler.cc

```diff
@@ -99,19 +101,23 @@ ss::future<topic_mount_result> topic_mount_handler::mount_topic(
       manifest, path_provider, parent);
     if (check_result != topic_mount_result::success) {
         vlog(
-          cst_log.error,
+          cst_log.warn,
```

Comment:
What is the motivation behind making these warnings instead of errors? I would imagine that someone going through the mount/unmount process would want these to be announced at the `error` level in case of failure in the logs.

Reply:
I don't have a strong opinion here, I suppose.